StalkStocks/StalkStocks
# Serves static files and fetches API data

Description
This request builds out the root handling of static files, as well as serves stock data pulledd from 3rd party API on navigation to /api

Asana Task
N/A

Testing Instructions
Run npm install to install dependancies, npm build to set up dist folder, and npm start to spin up server.  Localhost 3000 and 8000 will run on npm run dev as well.  Navigate or send a postman request to localhost:3000/api to see all incoming stock data.

Checklist
Please go through each item of this checklist carefully.

Additional Notes, Images, etc.
N/A
